lib/media/drm_engine.js

```diff
@@ -764,6 +778,10 @@ shaka.media.DrmEngine.prototype.processDrmInfos_ =
         }
       });
     }
+
+    if (drmInfo.defaultKeyID) {
```

Comment:
This is inconsistent with other places where you called it defaultKID, but as mentioned earlier, please rename to keyIds.